packages/uswds-core/src/js/utils/toggle-form-input.js

```diff
@@ -14,6 +13,19 @@ const HIDE_ATTR = "data-hide-text";
 const getHideText = (showText) =>
   showText.replace(/\bShow\b/i, (show) => `${show[0] === "S" ? "H" : "h"}ide`);
 
+const resolveIdRefs = (id) => {
+  const ids = id.trim().split(/\s+/);
+
+  return ids.map((i) => {
+    const el = document.getElementById(i);
+    if (!el) {
+      const err = `no element with id: ${i}`;
+      throw new Error(err);
+    }
+    return el;
+  });
+};
```

Comment:
I know it matches current behavior of `resolve-id-refs`, but I'd be curious if it's a good idea to interrupt code execution with a thrown error on a missing element, vs. silently ignoring (or `console.log`-ing) the missing element.

e.g.

```suggestion
const resolveIdRefs = (idRefs) =>
  idRefs
    .trim()
    .split(/\s+/)
    .map((id) => document.getElementById(id))
    .filter(Boolean);
```

Reply:
Thank you for looking! FWIW, I don't disagree at all. That said (and I recognize I'm probably being overly conservative!), for something we're trying to get in ASAP I'd prefer to leave the current behavior for now. This feature also doesn't have adequate test coverage. So what I'd prefer to do is to make an issue for your suggested improvements and write some tests such that next time something comes up here we can make changes more confidently.

Reply:
Sounds reasonable to me 👍